configs/neptune.yaml

```diff
@@ -35,7 +35,7 @@ parameters:
 # Execution
   clean_experiment_directory_before_training: 1
   num_workers: 4
-  num_threads: 1000
+  num_threads: 4
```

Comment:
@taraspiotr is it processes then ?

Reply:
I think its a ThreadPool but, a valid question is whats better. Normally id say processes should be better but since its mostly pytorch code might does not matter?

Reply:
@i008 @taraspiotr Lets's keep it as then